src/main/java/core/basesyntax/utils/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.utils;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        FigureType randomFigureType = getRandomFigureType();
+        double randomNumber = Math.round(random.nextDouble() * 10 + 1);
+        double randomNumber2 = Math.round(random.nextDouble() * 10 + 1);
+        double randomNumber3 = Math.round(random.nextDouble() * 10 + 1);
```

Comment:
this code is duplicating:
`Math.round(random.nextDouble() * 10 + 1)`
so its better to create method for it